src/client/automation/playback/scroll.js

```diff
@@ -54,8 +55,8 @@ export default class ScrollAutomation {
         var needForwardScrollTop  = y >= elementDimensions.scroll.top + elementDimensions.height;
         var needBackwardScrollTop = y <= elementDimensions.scroll.top;
 
-        var maxLeftScrollMargin = Math.min(MAX_SCROLL_MARGIN, Math.floor(elementDimensions.width / 2));
-        var maxTopScrollMargin  = Math.min(MAX_SCROLL_MARGIN, Math.floor(elementDimensions.height / 2));
+        var maxLeftScrollMargin = Math.min(MIN_SCROLL_MARGIN, Math.floor(elementDimensions.width / 2));
```

Comment:
Hmm, I don't understand what does it mean. What is it MIN_SCROLL_MARGIN and how `maxLeftScrollMargin` can be got by `Math.min` calling..